plugins/command-line/prism-command-line.js

```diff
@@ -54,6 +55,26 @@ Prism.hooks.add('complete', function (env) {
 	prompt.className = 'command-line-prompt';
 	prompt.innerHTML = lines;
 
+	var filterContent = pre.getAttribute('data-filter', '');
+	if (filterContent.length > 0) {
```

Comment:
You could use the `hasAttribute` method instead, to allow for empty values or absence of value too.

Reply:
Agreed.